sklearnex/ensemble/forest.py

```diff
@@ -16,3 +16,1192 @@
+            "removed in 1.2. Use `n_features_in_` instead.")
+        @property
+        def n_features_(self):
+            return self.n_features_in_
+
+    @property
+    def _estimators_(self):
+        if hasattr(self, '_cached_estimators_'):
+            if self._cached_estimators_:
+                return self._cached_estimators_
+        if sklearn_check_version('0.22'):
+            check_is_fitted(self)
+        else:
+            check_is_fitted(self, '_onedal_model')
+        classes_ = self.classes_[0]
+        n_classes_ = self.n_classes_[0]
+        # convert model to estimators
+        params = {
+            'criterion': self.criterion,
+            'max_depth': self.max_depth,
+            'min_samples_split': self.min_samples_split,
+            'min_samples_leaf': self.min_samples_leaf,
+            'min_weight_fraction_leaf': self.min_weight_fraction_leaf,
+            'max_features': self.max_features,
+            'max_leaf_nodes': self.max_leaf_nodes,
+            'min_impurity_decrease': self.min_impurity_decrease,
+            'random_state': None,
+        }
+        if not sklearn_check_version('1.0'):
+            params['min_impurity_split'] = self.min_impurity_split
+        est = ExtraTreeClassifier(**params)
+        # we need to set est.tree_ field with Trees constructed from Intel(R)
+        # oneAPI Data Analytics Library solution
+        estimators_ = []
+        random_state_checked = check_random_state(self.random_state)
+        for i in range(self.n_estimators):
+            est_i = clone(est)
+            est_i.set_params(
+                random_state=random_state_checked.randint(
+                    np.iinfo(
+                        np.int32).max))
+            if sklearn_check_version('1.0'):
+                est_i.n_features_in_ = self.n_features_in_
+            else:
+                est_i.n_features_ = self.n_features_in_
+            est_i.n_outputs_ = self.n_outputs_
+            est_i.classes_ = classes_
+            est_i.n_classes_ = n_classes_
+            tree_i_state_class = get_tree_state_cls(
+                self._onedal_model, i, n_classes_)
+            tree_i_state_dict = {
+                'max_depth': tree_i_state_class.max_depth,
+                'node_count': tree_i_state_class.node_count,
+                'nodes': tree_i_state_class.node_ar,
+                'values': tree_i_state_class.value_ar}
+            est_i.tree_ = Tree(
+                self.n_features_in_,
+                np.array(
+                    [n_classes_],
+                    dtype=np.intp),
+                self.n_outputs_)
+            est_i.tree_.__setstate__(tree_i_state_dict)
+            estimators_.append(est_i)
+
+        self._cached_estimators_ = estimators_
+        return estimators_
+
+    def _onedal_cpu_supported(self, method_name, *data):
+        _patching_status = PatchingConditionsChain(method_name)
+
+        if method_name == 'ensemble.ExtraTreesClassifier.fit':
```

Comment:
Method name is expected to be without module and class name starting from https://github.com/intel/scikit-learn-intelex/commit/5111fac5e07e9d7d46e330386546e823bbce98e7